ludwig/modules/convolutional_modules.py

```diff
@@ -1309,3 +1309,161 @@ def get_resnet_block_sizes(resnet_size):
             resnet_size, resnet_choices.keys()
         )
         raise ValueError(err)
+
+
+"""
```

Comment:
nit: Let's move this comment as a docstring on the class. Is there a paper you could add a link to?